pkg/sources/generator/tickgen.go

```diff
@@ -122,16 +121,14 @@ func WithReadTimeout(timeout time.Duration) Option {
 	}
 }
 
-// NewMemGen function creates an instance of generator.
```

Comment:
Why remove these docs?

Reply:
Because I think they are out of date. I can keep the first line though.